src/wallet/rpcwallet.cpp

```diff
@@ -497,8 +514,9 @@ static RPCHelpMan sendtoaddress()
 
     std::vector<CRecipient> recipients;
     ParseRecipients(address_amounts, subtractFeeFromAmount, recipients);
+    bool verbose = request.params[9].isNull() ? false: request.params[9].get_bool();
```

Comment:
micro-nit: spacing between `false:`

Reply:
for some reason git is ignoring my whitespace changes. Any suggestions on how to fix that?